tests/data/input-convert/simple2.asc

```diff
@@ -1,7 +1,10 @@
 ("CellBody"
  (Color Red)
  (CellBody)
- (0 0 0 2)
+ ( 0.1  0   0 2)
+ ( 0.1  0.1 0 2)
+ (-0.1  0   0 2)
+ (-0.1 -0.1 0 2)
```

Comment:
Isn't the shape of this soma weird?
Why not something like this?
```suggestion
 ( 0.1  0   0 2)
 ( 0  0.1 0 2)
 (-0.1  0   0 2)
 (0 -0.1 0 2)
```

Reply:
yeah, good  eye; it is weird - interestingly / unsurprisingly the tests didn't mind...
I'll prob. switch to something more sane

Reply:
done